src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -904,6 +932,38 @@ def argument(self, dest, arg_type=None, **kwargs):
         else:
             super(AzArgumentContext, self).argument(dest, arg_type=ignore_type)
 
+    def positional(self, dest, arg_type=None, **kwargs):
+        self._check_stale()
+        if not self._applicable():
+            return
+
+        if self.scope not in self.command_loader.command_table:
+            raise ValueError("command authoring error: positional argument '{}' cannot be registered to a group-level "
+                             "scope '{}'. It must be registered to a specific command.".format(dest, self.scope))
+
+        # Ensure max of 1 positional per command
+        command_args = self.command_loader.argument_registry.arguments[self.scope]
+        positional_args = {k: v for k, v in command_args.items() if not v.settings['options_list']}
+        if positional_args:
```

Comment:
Shouldn't this be `len(positional_args) > 1:`? This is not very obvious to me.

Reply:
Oh. If the change is indeed checked zero it's good. Just add your comments there. Because it is ambiguous to the reader.